ed-lti.php

```diff
@@ -25,7 +25,12 @@
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-require_once 'classes/class-ed-lti.php';
+namespace EdLTI;
+
+// Include the autoloader so we can dynamically include the rest of the classes.
+use EdLTI\classes\Ed_LTI;
```

Comment:
Does this use statement work? We haven't loaded the autoloader at this point so I thought it wouldn't do anything.

Reply:
seems to be fine